stream_alert/rule_processor/payload.py

```diff
@@ -252,7 +251,11 @@ def _get_object(self):
         LOGGER.debug('Pre-parsing record from S3. Bucket: %s, Key: %s, Size: %d',
                      bucket, key, self.s3_object_size)
 
-        return self._download_object(region, bucket, key)
+        try:
+            return self._download_object(region, bucket, key)
+        except IOError as err:
+            LOGGER.exception('[S3Payload] The following error occured while downloading: \n%s', err)
```

Comment:
`LOGGER.exception` will log the contents of the err, so you can omit the explicit 'err' inclusion since it will be redundant. For instance:

```
try:
    return self._download...(...)
except IOError:
    LOGGER.exception('[S3Payload] The following error occurred while downloading")
    return
```
Also there's a typo: `occured` --> `occurred`